db/models.py

```diff
@@ -1,4 +1,7 @@
+from django.contrib.auth.models import AbstractUser
+from django.core.exceptions import ValidationError
 from django.db import models
+from settings import AUTH_USER_MODEL
```

Comment:
The import statement for `AUTH_USER_MODEL` is incorrect. It should be `from django.conf import settings` and then use `settings.AUTH_USER_MODEL` in the `Order` model.